Tests/test_SeqIO_Gck.py

```diff
@@ -380,6 +380,20 @@ def test_conflicting_lengths(self):
         h.close()
 
 
+class TestGckWithImproperHeader(unittest.TestCase):
+
+    def setUp(self):
+        with open('Gck/improperHeader.gck', 'rb') as f:
+            self.buffer = f.read()
+
+    def test_read(self):
+        """Read a file with an incomplete header."""
+        h = BytesIO(self.buffer)
```

Comment:
I would remove the ``setUp`` here and just do the open directly in this test method.

Reply:
(This also avoids the old fashioned open and close - just use the with statement)